src/groups/groups.service.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { Sequelize } from 'sequelize';
+
+import { CreateGroupDto } from './dto/create-group.dto';
+import { UpdateGroupDto } from './dto/update-group.dto';
+import { groups } from './entities/group.entity';
+import {
+  permissions_groups,
+  permissions,
+} from '../permissions/entities/permissions.entity';
+
+@Injectable()
+export class GroupsService {
+  constructor(
+    @InjectModel(groups)
+    private groupsModel: typeof groups,
+    @InjectModel(permissions)
+    private permissionsModel: typeof permissions,
+    @InjectModel(permissions_groups)
+    private permissionsGroupsModel: typeof permissions_groups,
+    private sequelize: Sequelize,
+  ) {}
+
+  async create(createGroupDto: CreateGroupDto) {
+    const t = await this.sequelize.transaction();
```

Comment:
Could you please give to `t` more readable name?

Reply:
fixed